.ci/pytorch/test.sh

```diff
@@ -692,7 +692,7 @@ test_xpu_bin(){
 
   for xpu_case in "${BUILD_BIN_DIR}"/*{xpu,sycl}*
   do
```

Comment:
Please note, no other tests in this file have `do` on a newline
```suggestion
  for xpu_case in "${BUILD_BIN_DIR}"/*{xpu,sycl}*; do
```